### PR TITLE
WIP - Feature/logins 4

### LINKED ISSUE #4: logins
As a CC staff member, I can log in, ensuring that only authorised people can then do some functions

Acceptance
- [x] Seed the first user
- [x] Email the user a link to set password
- [x] email and password to log in
- [x] Ability to reset your password by email

### PR DESCRIPTION
**Pull request by Br3nda**
# What has changed and why?

# How to test this change

- [x] has automated tests
- [x] at least one a reviewer ran the code
- [x] updated API docs

closes #4 

- [x] Seed the first user
- [x] Email the user a link to set password
- [x] email and password to log in
- [x] Ability to reset your password by email

Also scaffolded viewing rebate forms, so there's something to see behind the logins

### CODE REVIEW

File: app/controllers/admin/rebate_forms_controller.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class Admin::RebateFormsController < ApplicationController
+  before_action :set_admin_rebate_form, only: %i[show edit update destroy]
+
+  # GET /admin/rebate_forms
```
Comment (dlouise64) [inline app/controllers/admin/rebate_forms_controller.rb]: @Br3nda whats going to go here? is that submitted form data?

File: app/controllers/users/passwords_controller.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  # def new
+  #   super
+  # end
+
+  # POST /resource/password
+  # def create
+  #   super
+  # end
+
+  # GET /resource/password/edit?reset_password_token=abcdef
```
Comment (dlouise64) [inline app/controllers/users/passwords_controller.rb]: @Br3nda there is a checkbox for password reset in the review template, but I haven't tested this yet.

File: config/environments/development.rb [hunk 3]
```diff
@@ -32,6 +32,7 @@
   config.action_mailer.raise_delivery_errors = false
 
   config.action_mailer.perform_caching = false
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```
Comment (dlouise64) [inline config/environments/development.rb]: @Br3nda we could change the port to something like `Rails::Server.new.options[:Port]`sometimes Im running multiple instances or react on port 3000, this should get the current port.

File: app/controllers/admin/rebate_forms_controller.rb (hunk 1 above)
Comment (Br3nda) [reply app/controllers/admin/rebate_forms_controller.rb]: It's controller for CRUD on rebate when logged into admin.

**Pull request merged**